content/_help/get-started/authentication-options._fr.md

```diff
@@ -16,25 +16,14 @@ redirect_from:
   - /fr/help/authentication-methods/which-authentication-method-should-i-use/
   - /fr/help/authentication-methods/
 ---
-En plus de votre mot de passe, Login.gov exige que vous mettiez en place au moins une méthode d'authentification secondaire pour assurer la sécurité de votre compte. Il s'agit d'une authentification à deux facteurs (A2F). Nous utilisons l'A2F comme un niveau de protection supplémentaire pour sécuriser vos informations.
+En plus de votre mot de passe, Login.gov exige que vous mettiez en place au moins une méthode d'authentification pour assurer la sécurité de votre compte. Il s'agit d'une authentification multifactorielle (AMF). Nous utilisons l'AMF comme un niveau de protection supplémentaire pour sécuriser vos informations.
 
 **Authentification secondaire**
```

Comment:
One more: The other translations removed references to "secondary". Are we doing that for French as well?

Reply:
Sorry for all the thrash. Will pay closer attention moving forward / document changes more easily. Wish Netlify showed ES and FR side by side with the EN ...